pkg/api/errors/errors.go

```diff
@@ -226,7 +226,7 @@ func NewServerTimeout(kind, operation string, retryAfterSeconds int) error {
 		Details: &unversioned.StatusDetails{
 			Kind:              kind,
 			Name:              operation,
-			RetryAfterSeconds: retryAfterSeconds,
+			RetryAfterSeconds: int32(retryAfterSeconds),
```

Comment:
I am guessing the narrowing/widening conversions here are because you don't want to change the API all over town, or do you really prefer the API for this to be an int.